clusterloader2/testing/experimental/storage/pod-startup/persistent-volumes/config.yaml

```diff
@@ -0,0 +1,142 @@
+# ASSUMPTIONS:
+# TODO: Assumptions
+
+# Constants
+  {{$NODE_MODE := DefaultParam .NODE_MODE "allnodes"}}
+  {{$DEFAULT_NODES_PER_NAMESPACE := DefaultParam .NODES_PER_NAMESPACE 100}}
+
+# Variables
+  {{$PODS_PER_NODE := DefaultParam .PODS_PER_NODE 50}}
+  {{$PVC_PER_POD := DefaultParam .PVC_PER_POD 2}}
+  {{$ENABLE_CHAOSMONKEY := DefaultParam .ENABLE_CHAOSMONKEY false}}
+  {{$TEST_NAME := DefaultParam .TEST_NAME "generic-test"}}
+  {{$PVC_SIZE := DefaultParam .PVC_SIZE "8Gi"}}
+  {{$PER_PVC_BOUND_SLO := DefaultParam .PVC_BOUND_SLO 2}}
```

Comment:
is this in seconds? 2 seconds binding seems low. Not sure if this includes provisioning time or not though

Reply:
Provisioning is done in parallel to a degree. 2 sec is low if you are no provisioning that many, which is why I use a PVCBoundTime of min 60 sec.

Reply:
sorry still confuse about the meaning of this parameter and the set of "2"?

Reply:
It just means that because the backend can only provision a certain number of PVs at a time and thus is to a degree serial, I set an SLO for the per PV provision time at 2 seconds. So 100 PVs has a total SLO of 2 * 100 = 200 seconds.

Reply:
I think 2 seconds is too fast, especially while we're in an experimental phase and are testing our limits. Let's do something like 30 seconds?

We're also limited by qps of k8s api server and gce api